src/index.test.js

```diff
@@ -148,3 +162,59 @@ test('deactivates dropdown active on blur', t => {
   wrapper.instance().onInputBlur()
   t.false(wrapper.instance().keepDropdownActive)
 })
+
+test('detects click outside', t => {
```

Comment:
Thanks for adding this, was much needed! 🙏

Reply:
Np. Was not easily fixed without adding any new dependency since dispatchEvent/simulate does not seem to bubble to the virtual jsdom document eventlistener. Had trigger it manually. Maybe could be fixed better if adding simulant.js.